.rubocop.yml

```diff
@@ -41,6 +41,8 @@ Metrics/MethodLength:
   CountComments: false
   Max: 20
   Severity: error
+  Exclude:
+    - lib/jekyll/command.rb
```

Comment:
> You can split the def `add_build_options` into smaller methods

Nope. Not doing that.

Just exclude this one method from the `MethodLength` cop.

```ruby
# rubocop:disable Metrics/MethodLength
def add_build_options(c)
    ....
end
# rubocop:enable Metrics/MethodLength